### PR TITLE
Finish User Edit Profile

### PR DESCRIPTION
**Pull request by hitshootnum1**

### CODE REVIEW

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -4,12 +4,23 @@
...
     </div>
     <ul class="nav navbar-nav">
-      <li><%= link_to t("layouts.home"), root_path %></li>
       <% if user_signed_in? %>
-        <li>
-          <%= link_to t("layouts.log_out"), destroy_user_session_path,
-            method: :delete %>
-        </li>
+        <li class="dropdown">
+            <a href="#" class="dropdown-toggle" data-toggle="dropdown">
```
Comment (QuangVinh93) [inline app/views/layouts/_header.html.erb]: thay thẻ <a> bằng link_to

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -4,12 +4,23 @@
...
+              Account <b class="caret"></b>
```
Comment (QuangVinh93) [inline app/views/layouts/_header.html.erb]: I18n

**Comment by QuangVinh93** [comment]
ok

**Comment by HUSTToanLH** [comment]
ok

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -4,12 +4,24 @@
...
     </div>
     <ul class="nav navbar-nav">
-      <li><%= link_to t("layouts.home"), root_path %></li>
       <% if user_signed_in? %>
-        <li>
-          <%= link_to t("layouts.log_out"), destroy_user_session_path,
-            method: :delete %>
-        </li>
+        <li class="dropdown">
+            <%= link_to "#", class: "dropdown-toggle",
```
Comment (dieunb) [inline app/views/layouts/_header.html.erb]: indent

**Pull request merged**